set.mm

```diff
@@ -32528,13 +32528,25 @@ general is seen either by substitution (when the variable ` V ` has no
 
   ${
     $d A y $.  $d x y $.
+    $( Closed theorem form of ~ isset that does not require ` x ` and ` A ` to
+       be distinct.  Extracted from the proof of ~ vtoclgft .  (Contributed by
+       Wolf Lammen, 9-Apr-2025.) $)
+    issetft $p |- ( F/_ x A -> ( A e. _V <-> E. x x = A ) ) $=
+      ( vy cvv wcel cv wceq wex wnfc isset wn wal nfv nfnfc1 nfcvd id nfnd nfvd
+      nfeqd alnex weq wb wi eqeq1 notbid a1i cbv2w 3bitr3g con4bid bitrid ) BDE
+      CFZBGZCHZABIZAFZBGZAHZCBJUNUMUQUNULKZCLUPKZALUMKUQKUNURUSCAUNCMABNUNULAUN
+      AUKBUNAUKOUNPSQUNUSCRCAUAZURUSUBUCUNUTULUPUKUOBUDUEUFUGULCTUPATUHUIUJ $.
+
     issetf.1 $e |- F/_ x A $.
     $( A version of ~ isset that does not require ` x ` and ` A ` to be
        distinct.  (Contributed by Andrew Salmon, 6-Jun-2011.)  (Revised by
        Mario Carneiro, 10-Oct-2016.) $)
     issetf $p |- ( A e. _V <-> E. x x = A ) $=
-      ( vy cvv wcel cv wceq wex isset nfeq2 nfv eqeq1 cbvexv1 bitri ) BEFDGZBHZ
-      DIAGZBHZAIDBJQSDAAPBCKSDLPRBMNO $.
+      ( wnfc cvv wcel cv wceq wex wb issetft ax-mp ) ABDBEFAGBHAIJCABKL $.
+  $}
+
+  ${
+    $d A y $.  $d x y $.
   $}
```

Comment:
Phantom DV conditions.

Reply:
y is a dummy variable used during the proof (z in the original in vtoclgft) that needs these conditions

Reply:
OK you don't need them for issetf any more.

Reply:
There seems to be a misundertanding. In your branch of set.mm there are these three lines:

<pre>
${
  $d A y $.  $d x y $.
$}
</pre>

These DV conditions are not used by any theorem, so they are phantom. You can just remove these three lines (your new commit 084e240 does not fix this.)

Reply:
It's good now.